src/styles/_tmp-variables.scss

```
@@ -3,5 +3,16 @@
 
 $tmp-animation-timing-2: 0.2s;
 
+$tmp-color-green-100: #e8f8f5;
```

suggestion: it might make sense to add some comment headers for different "sections" in this file

```suggestion
// Colors 
$tmp-color-green-100: #e8f8f5;
```

---

Done